src/components/Modal/modal.module.scss

```diff
@@ -10,7 +10,7 @@
     }
 }
 .modalOverlay {
-    z-index: 9999;
+    z-index: 9998;
```

Comment:
But why is soo large Z index needed?

Reply:
Can you please create a seperate issues to fix the z index values, having random large values will create lots of issues in future